<?xml version="1.0"?>
<launch>
  <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="joy" output="screen">
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.1" />
  </node>
	
  <!-- Joystick Teleop Node -->
  <node pkg="youbot_teleop" type="youbot_teleop" name="youbot_teleop" output="screen">
    <rosparam command="load" file="$(find youbot_teleop)/config/params.yaml" />
    <rosparam command="load" file="$(find aricc_rcwork_task)/config/arm_joints.yaml" />
    <!-- Axes -->
    <param name="axis_linear_x" value="1" type="int" />
    <param name="axis_linear_y" value="0" type="int" />
    <param name="axis_angular"  value="2" type="int" />
    <!-- Scale -->
    <param name="scale_linear" value="0.3" type="double" />
    <param name="scale_angular" value="0.3" type="double" />
    <remap from="cmd_vel" to="cmd_vel_teleop"/> 
  </node>
   
</launch>
